lib/src/speed_dial_direction.dart

```diff
@@ -0,0 +1,6 @@
+enum SpeedDialDirection { Up, Down, Left, Right }
+
+extension EnumExtension on SpeedDialDirection {
+  /// Get Value of The SpeedDialDirection Enum like Up, Down, etc. in String format
+  String get value => this.toString().split(".")[1];
```

Comment:
This functionality is already in flutter: https://api.flutter.dev/flutter/foundation/describeEnum.html

Reply:
Yes, but I liked this way cause it doesn't require importing any library.